grpc/src/main/java/com/linecorp/armeria/internal/grpc/ArmeriaMessageDeframer.java

```diff
@@ -267,6 +267,11 @@ public boolean isClosed() {
         return unprocessed == null;
     }
 
+    public ArmeriaMessageDeframer decompressor(Decompressor decompressor) {
+        this.decompressor = decompressor;
```

Comment:
Could we add some state check so that it's not allowed to change the decompressor if 1) this method was called before or 2) deframing started already?